easybuild/easyconfigs/a/assimp/assimp-5.4.3-GCCcore-13.3.0.eb

```diff
@@ -0,0 +1,36 @@
+# Authors:: Richard Lawrence  <rarensu@tamu.edu> - TAMU HPRC - https://hprc.tamu.edu
+
+easyblock = 'CMakeMake'
+
+name = 'assimp'
+version = '5.4.3'
+
+homepage = 'https://github.com/assimp/assimp'
+description = """
+ Open Asset Import Library (assimp) is a library to import and export various
+ 3d-model-formats including scene-post-processing to generate missing render data.
+"""
+
+toolchain = {'name': 'GCCcore', 'version': '13.3.0'}
+
+source_urls = ['https://github.com/%(name)s/%(name)s/archive']
+sources = ['v%(version)s.tar.gz']
+checksums = ['66dfbaee288f2bc43172440a55d0235dfc7bf885dda6435c038e8000e79582cb']
+
+builddependencies = [
+    ('binutils', '2.42'),
+    ('pkgconf', '2.2.0'),
+    ('CMake', '3.29.3'),
+    ('zlib', '1.3.1'),
+]
+
+# workaround bug with GCC13 https://github.com/assimp/assimp/issues/5315
+configopts = "-DASSIMP_WARNINGS_AS_ERRORS=OFF "
+
+
+sanity_check_paths = {
+    'files': ['include/%(name)s/types.h', 'lib/libassimp.so'],
```

Comment:
```suggestion
    'files': ['include/%(name)s/types.h', 'lib/libassimp.%s' % SHLIB_EXT],
```

Reply:
thanks, indeed, it was changed by `--copy-ec` option :(